src/tests/db.test.ts

```diff
@@ -42,7 +42,7 @@ describe("user test cases", () => {
         where: { taskId: t.id },
       });
 
-      expect(userTasks.length).toBeGreaterThan(0);
+      expect(userTasks.length);
```

Comment:
I think you removed the assert here

```suggestion
      expect(userTasks.length).toHaveLength(0);
```

Reply:
expect(userTasks).toHaveLength(0)